src/core/config.js

```diff
@@ -91,11 +91,22 @@ if ( urlParams.seed ) {
 }
 
 config.testId = [];
+config.moduleId = [];
+var i;
 if ( urlParams.testId ) {
 
 	// Ensure that urlParams.testId is an array
 	urlParams.testId = decodeURIComponent( urlParams.testId ).split( "," );
-	for ( var i = 0; i < urlParams.testId.length; i++ ) {
+	for ( i = 0; i < urlParams.testId.length; i++ ) {
 		config.testId.push( urlParams.testId[ i ] );
 	}
 }
+
+if ( urlParams.moduleId ) {
+
+	// Ensure that urlParams.moduleId is an array
+	urlParams.moduleId = decodeURIComponent( urlParams.moduleId ).split( "," );
+	for ( i = 0; i < urlParams.moduleId.length; i++ ) {
+		config.moduleId.push( urlParams.moduleId[ i ] );
+	}
+}
```

Comment:
Since this logic is identical to the logic for `testId`, could we abstract this into a function? Maybe something like:

```js
config.moduleId = getIds(urlParams.moduleId);
```